torch/_refs/__init__.py

```diff
@@ -415,16 +439,19 @@ def _ref(a: TensorLikeType) -> TensorLikeType:
     return inner
 
 
-def _make_alias(fn, name):
+def _make_alias(fn, name, module_name):
     """
-    This function defines an alias of another function and sets its __name__argument
-    Note that when naïvely doing `alias = fn`, we have that `alias.__name__ == "fn"`.
+    This function defines an alias of another function and sets its __name__ and __module__
+    argument.
+    Note that when naïvely doing `alias = fn`, we have that `alias.__name__ == "fn"`, and
+    `alias.__module__ == fn.__module__`.
     """
 
     def _fn(*args, **kwargs):
         return fn(*args, **kwargs)
 
     _fn.__name__ = name
+    _fn.__module__ = module_name
```

Comment:
If you want to do this automatically, you can inspect at the module fo the previous frame. It'd allow for a neater and less error-prone API at the cost of one magic trick.

Reply:
okay, I was originally thinking the same but worried it would be considered too hacky.